### PR TITLE
[TASK] Use bootstrap-package 12.0

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

**Comment by linawolf** [comment]
My-Manual läuft problemlos durch, können wir den mergen?

File: composer.json [hunk 1]
```diff
@@ -50,6 +54,7 @@
 		"typo3/cms-indexed-search": "dev-master as 11.3.1",
 		"typo3/cms-info": "dev-master as 11.3.1",
 		"typo3/cms-install": "dev-master as 11.3.1",
+		"typo3/cms-introduction": "dev-master",
```
Comment (alexander-nitsche) [inline composer.json:57]: composer.json's 
```
"typo3/cms-introduction": "dev-master"
```
does not match with composer.lock's
```
"typo3/cms-introduction": "dev-BP_12-0"
```

File: composer.json [hunk 2]
```diff
@@ -26,11 +22,19 @@
 			"url": "./packages/*"
 		}
 	],
+	"minimum-stability": "dev",
```
Comment (alexander-nitsche) [inline composer.json:25]: Is this change in minimum stability required? we worked with `dev-master` already without this requiring to lower the stability here. consequence of it seems to be that all dependencies also get downloaded in development version. not sure if you want achieve that.

**Review by alexander-nitsche — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (linawolf) [reply composer.json:57]: I think you are mixing up something here, cms-introduction ist also as dev-master in the composer.lock

It is the bootstrap package we need as 
            "name": "bk2k/bootstrap-package",
            "version": "dev-BP_12-0",

File: composer.json (hunk 2 above)
Comment (linawolf) [reply composer.json:25]: Without it I get the following error:

```
  Problem 1
    - Root composer.json requires typo3/cms-introduction dev-master -> satisfiable by typo3/cms-introduction[dev-master].
    - typo3/cms-introduction dev-master requires bk2k/bootstrap-package dev-BP_12-0 -> found bk2k/bootstrap-package[dev-BP_12-0, 12.0.x-dev (alias of dev-BP_12-0)] but it does not match your minimum-stability.
```

Comment (linawolf) [reply composer.json:25]: Got it removed

**Review by alexander-nitsche — APPROVED** [review]
Looks better now.

**Pull request merged**